src/components/shared/card.tsx

```diff
@@ -138,7 +149,10 @@ const cardStyles = (itemType: RelatedItemType, format: Format): SerializedStyles
         }
 
         case RelatedItemType.ADVERTISEMENT_FEATURE: {
-            return css``;
+            return css`
+                background-color : ${neutral[97]};
```

Comment:
I think these look a little darker on apps, maybe neutral[93]?

Reply:
yeah at the moment `97` is used for the background of relatedContent so these cards will blend in

Reply:
Agreed, on the app the background on which all the cards sit is white, however in 'local browser' view it is grey [93] so it will blend in to the bg - in short yes the app its white and [93] for the bg of the card is better 👍